pkg/test/resourcefixture/testdata/basic/compute/v1alpha1/computemanagedsslcertificate/managedsslcertificate/create.yaml

```diff
@@ -11,15 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 apiVersion: compute.cnrm.cloud.google.com/v1alpha1
 kind: ComputeManagedSSLCertificate
 metadata:
-  name: computemanagedsslcertificate-${uniqueId}
+  name: computemanagedsslcertificate${uniqueId}
 spec:
   managed:
-  - domains:
-    - sslcert.tf-test.club.
+    domains:
+    - sslcert.kcc-test.club.
```

Comment:
can you change this to `direct` since we try to avoid exposing these codenames (I know... we refer them a lot in comments and PR descriptions. But we want to not expose them in the actual codebase.